services/tsd-api-mock/src/main/java/no/elixir/tsdapimock/resumables/ResumableUploadDto.java

```diff
@@ -11,4 +11,4 @@ public record ResumableUploadDto(
     @JsonProperty("previous_offset") BigInteger previousOffset,
     @JsonProperty("next_offset") BigInteger nextOffset,
     @JsonProperty("max_chunk") BigInteger maxChunk,
-    @JsonProperty("md5sum") String md5Sum) {}
+    @JsonProperty("md5Sum") String md5Sum) {}
```

Comment:
As commented above, are we sure we mimic the correct behaviour in the realworld TSD API here?

Reply:
Temporary fix with [eb97539](https://github.com/ELIXIR-NO/FEGA-Norway/pull/189/commits/eb97539c643fa3d77913c28ac5145d7ca1172418)